pages/gallery.html

```diff
@@ -1,12 +1,60 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
-  <meta charset="UTF-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+	<meta charset="UTF-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<link rel="stylesheet" href="/styles/styles.css">
+	<link rel="stylesheet" href="/styles/gallery.css">
+	<title>Gallery</title>
 </head>
+
 <body>
-  
+	<nav class="nav">
+		<h1>Janice and Ericka's Grey's Anatomy Fan Page!</h1>
+		<h3>Gallery</h3>
+		<a class="active" href="/pages/index.html">Home</a>
+		<a href="/pages/facts.html" rel="import">Fun Facts</a>
+		<a href=/pages/gallery.html rel="import">Gallery</a>
+	</nav>
+	<section class="image-container">
+		<div class="item1">
+			<img src="/images/meredith-grey.jpg" alt="Meredith Grey">
+			<p class="character">Meredith Grey</p>
+		</div>
+		<div class="item2">
+			<img src="/images/cristina_yang.jpg" alt="Cristina Yang">
+			<p class="character">Christina Yang</p>
+		</div>
```

Comment:
Considering how all these items have identical structure, I would really recommend using that to your advantage.

What if the div had a reused class, like `character-item` instead of a unique `item1` class?

Then, what if instead of `class="character"` on each paragraph, you could select the paragraph child of any element with the `character-item` class?